contact.php

```diff
@@ -120,11 +120,18 @@
 
 $getemail = '';
 $ergebnis = safe_query("SELECT * FROM `" . PREFIX . "contact` ORDER BY `sort`");
-while ($ds = mysqli_fetch_array($ergebnis)) {
+if(mysqli_num_rows($ergebnis)<1) {
```

Comment:
Switch if-else so that the positive event is the if and the negative event is the else.

```
if(mysqli_num_rows($ergebnis)) {
// we have results - do something with them
} else {
// no results, show error
}
```